web/test/data/repositories/repository-minimum.json

```diff
@@ -0,0 +1,12 @@
+{
+  "uid": "test-repository-minimum",
+  "projects": ["test-project-public-repository"],
+  "url": "https://api.github.com/repos/webkhushboo/HelloWorld",
```

Comment:
Probably best to use DashboardHub project here

Reply:
ok but here I have to validate for no data scenario